cli/cmd/generate_terraform_test.go

```diff
@@ -0,0 +1,215 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test various scenarios for detection and/or installation of Terraform binary
+func TestGenerationTfInstallation(t *testing.T) {
+	if os.Getenv("CI") == "true" {
```

Comment:
Can we change the logic of this `if` statement and use `t.Skip()` instead?
```suggestion
	if os.Getenv("CI") != "" {
		t.Skip(fmt.Sprintf("skipping %s", t.Name()))
	}
```

Reply:
👍